tsconfig.json

```diff
@@ -20,6 +21,7 @@
     "strictNullChecks": true,
     "strictPropertyInitialization": false,
     "target": "ES6",
+    "types": ["reflect-metadata"],
```

Comment:
Are we sure this doesn't cause any issues? We know that the issue was not related to this.

Reply:
Yeah will rollback